tox.ini

```diff
@@ -2,6 +2,7 @@
 envlist = packaging, py35, py36, py37, py38, py39, check_codestyle, check_isort
 
 [base]
+download = False
```

Comment:
what does this do? I don't think adding entries to `base` does anything unless it gets referenced from another section?